lib/Runtime/Library/GlobalObject.cpp

```diff
@@ -599,10 +599,17 @@ namespace Js
         if (!scriptContext->IsInEvalMap(key, isIndirect, &pfuncScript))
         {
             uint32 grfscr = additionalGrfscr | fscrReturnExpression | fscrEval | fscrEvalCode | fscrGlobalCode;
+
             if (isLibraryCode)
             {
                 grfscr |= fscrIsLibraryCode;
             }
+
+            if (isIndirect)
+            {
+                grfscr |= fscrIndirect;
+            }
+
             pfuncScript = library->GetGlobalObject()->EvalHelper(scriptContext, argString->GetSz(), argString->GetLength(), moduleID,
```

Comment:
>EvalHelper [](start = 54, length = 10)

you are already updating grfscr above, so what is the point of passing the isIndirect bool to the function EvalHelper ?